stories/components/section/section.html.twig

```diff
@@ -20,9 +20,8 @@
       <div class="bix-section__body">
         {{ description | raw }}
       </div>
-      {% else %}
-      {{ body }}
       {% endif %}
+      {{ body }}
```

Comment:
Refactor section so that body and description are completely separate. How this currently is is messed up in story book on the 'About us' page with _people-section.html.twig because it tries to use description AND set the body block which breaks it.
Sections are nested in storybook for the _core-services.html.twig which shows the flip card and a section underneath it on the homepage.
That one uses ONLY body.
Both those places should still work after this change.
Possible name changes:
Block body is now block description
Variable body is now variable nested_sections.
In core-services.html.twig, change the {% block body %} -> {{ set nested_sections }} {{endset}}